src/scripts/release.zig

```diff
@@ -58,6 +59,10 @@ const multiversion_epoch = "0.15.3";
 /// This commit references https://github.com/tigerbeetle/tigerbeetle/pull/1935.
 const multiversion_epoch_commit = "035c895bf85f5106d94f08cac49719994344880e";
 
+/// As does this tag. Have it as an explicit tag, so it's not just a random commit SHA that's
```

Comment:
I'd just copy the `This commit references https://github.com/tigerbeetle/tigerbeetle/pull/1935.` comment down, to be more robust to future shuffling of these lines.